src/common/consensus-provider/consensus-provider.service.ts

```diff
@@ -290,10 +290,19 @@ export class ConsensusProviderService {
       return cached.missed ? undefined : cached.info;
     }
 
+    const workingMode = this.config.get('WORKING_MODE');
+
     const blockInfo = await this.retryRequest<BlockInfoResponse>(
       async (apiURL: string) => this.apiGet(apiURL, this.endpoints.blockInfo(blockId)),
       {
         maxRetries: this.config.get('CL_API_GET_BLOCK_INFO_MAX_RETRIES'),
+        useFallbackOnResolved: (r) => {
+          if (r.hasOwnProperty('finalized') && !r.finalized && workingMode === WorkingMode.Finalized) {
+            this.logger.error(`State for slot ${r.data.message.slot} is not finalized`);
```

Comment:
```suggestion
            this.logger.error(`Slot [${r.data.message.slot}] is not finalized`);
```
just to not mislead by the `State` term because it is already used in the logs as `state endpoint` and similar

Reply:
OK, I've changed the log messages.